tools/configure.py

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 # Lint as: python3
 
+"""Tools that implements vendor-specific CTAP2 commands to configure OpenSK.
+"""
```

Comment:
Is this one-line doc supposed to have it's closing """ on the second line?

Also: Tools that implements -> Tools that implement

Reply:
I thought that might be the case, but the limit must be slightly smaller, because:

```
$ len('"""Tools that implement vendor-specific CTAP2 commands to configure OpenSK."""')
78
```